ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -74,7 +74,7 @@ function build_push_operator_images {
   if [[ -f storage-bundle.Dockerfile ]]; then
     DOCKERFILE=storage-bundle.Dockerfile /bin/bash hack/pin-custom-bundle-dockerfile.sh
```

Comment:
NOTE: This needs to be set here so it passes through the subshell /bin/bash command. It does not get applied below where I set it directly to the .pinned version which is generated by this command.